linked-list.rb

```diff
@@ -63,9 +64,43 @@ def display
    end
 
    def include?(key)
+
+     # Traverse through the list till you hit the "nil" at the end
+     current = @head
+     full_list = []
+     while current.next_node != nil
+         if current.value == key
+           return true
+         else
+           full_list += [current.value.to_s]
+           current = current.next_node
+         end
+     end
+
+     if current.value == key
+       return true
+     else
+       full_list += [current.value.to_s]
+     end
+
+     return false
+
    end
```

Comment:
This looks generally good except that we don't need any of the lines with `full_list` because we're just looking for a value in the list and then returning true or false if we find it or not